src/language_concepts/data_types.md

```diff
@@ -1,206 +1,192 @@
-# Data types
+# Data Types
 
-Noir has several varieties of data types: the primitive `Field` type, integer types, `bool`, arrays, structs, and tuple types.
+Every value in Noir is specified with a certain data type, which informs Noir tools how to work with that data when processing Noir programs.
```

Comment:
Lets reword this. "is specified with" implies types are required to be specified for all values rather than being inferred. We can say "Every value in noir has a type which determines which operations are valid for it." The point on tooling seems unnecessary especially since the only tooling that looks at types currently is the compiler itself.